### ISSUE TITLE
Figured bass

### ISSUE THREAD

**Issue by cbmajor7** [artusiinc/issues#61]
It would be great to be able to use figured bass.

**Comment by mscuthbert** [comment]
Yep!  We're working on it and having fun with it on the dev team -- it's just a little bit too buggy to release to students yet.
